### PR TITLE
feat: use trustwallet instead of coincap for the base tokens icons and use bigger icons

### LINKED ISSUE #7258: token icons from coingecko
goal: remove coincap as single point of failure 

- remove coincap
- update asset generation to use large icons from coingecko
- if needed suggest viable alternative to coingecko if we find edge cases.

### PR DESCRIPTION
**Pull request by NeOMakinG**
## Description

I replaced coincap token icons in favor of the trustwallet repository, I also changed them inside tests and mocks so mocks are linked to real datas as much as possible

Most of the icons were already using coingecko except for the base assets. Coingecko is really often not up to date, icons are wrong, for the base assets trustwallet seems like a more reliable solution!

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #7258

## Risk
Low as it's mainly presentational

## Testing
- Rebuild the assets by using `yarn generate:asset-data`
- Launch the app and verify that the base assets are rightly displayed, also check for FOX, rFOX

### Engineering
n/a

### Operations
n/a

## Screenshots (if applicable)
image

### CODE REVIEW

File: scripts/generateAssetData/generateAssetData.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ const generateAssetData = async () => {
     name: 'FOX on Arbitrum One',
     precision: 18,
     color: '#3761F9',
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```
Comment (kaladinlight) [inline scripts/generateAssetData/generateAssetData.ts:177]: can we replace all references to `FOX` icons with the large icon suggested here please
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

File: src/contracts/constants.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { getConfig } from 'config'
+import { getConfig } from '../config'
```
Comment (kaladinlight) [inline src/contracts/constants.ts:1]: revert me

File: scripts/generateAssetData/baseAssets.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ export const ethereum: AssetWithNetworkName = {
   networkName: 'Ethereum',
   precision: 18,
   color: '#5C6BC0',
-  icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
+  icon: '',
```
Comment (kaladinlight) [inline scripts/generateAssetData/baseAssets.ts:14]: update all references to ethereum icon to the one suggested here please 
```suggestion
  icon: 'https://coin-images.coingecko.com/coins/images/279/large/ethereum.png?1696501628',
```

File: scripts/generateAssetData/ethereum/uniswapV2Pools.ts [hunk 4]
```diff
@@ -105,7 +105,7 @@ export const getUniswapV2Pools = async () => {
         name,
         precision: Number(appTokenData.decimals),
         icons,
-        icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
+        icon: 'https://assets.coingecko.com/coins/images/12504/standard/uni.jpg?1696512319',
```
Comment (kaladinlight) [inline scripts/generateAssetData/ethereum/uniswapV2Pools.ts:108]:
```suggestion
        icon: 'https://coin-images.coingecko.com/coins/images/12504/large/uni.jpg?1696512319',
```

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 5]
```diff
@@ -100,13 +100,13 @@ export const STAKING_ID_DELIMITER = '*'
 export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadata> = {
   [DefiProvider.ShapeShift]: {
     provider: DefiProvider.ShapeShift,
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```
Comment (kaladinlight) [inline src/state/slices/opportunitiesSlice/constants.ts:103]:
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 6]
```diff
@@ -100,13 +100,13 @@ export const STAKING_ID_DELIMITER = '*'
...
     color: '#3761F9',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.EthFoxStaking]: {
     provider: DefiProvider.EthFoxStaking,
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```
Comment (kaladinlight) [inline src/state/slices/opportunitiesSlice/constants.ts:109]:
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 7]
```diff
@@ -118,13 +118,13 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
   },
   [DefiProvider.CosmosSdk]: {
     provider: DefiProvider.CosmosSdk,
-    icon: 'https://assets.coincap.io/assets/icons/256/atom.png',
+    icon: 'https://assets.coingecko.com/coins/images/1481/standard/cosmos_hub.png?1696502525',
```
Comment (kaladinlight) [inline src/state/slices/opportunitiesSlice/constants.ts:121]:
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/1481/large/cosmos_hub.png?1696502525',
```

File: src/contracts/constants.ts (hunk 2 above)
Comment (NeOMakinG) [reply src/contracts/constants.ts:1]: @kaladinlight are you able to build the assets locally without this? I wasn't

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 8]
```diff
@@ -118,13 +118,13 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
...
     color: '#C5B5F2',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.ThorchainSavers]: {
     provider: DefiProvider.ThorchainSavers,
-    icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+    icon: 'https://assets.coingecko.com/coins/images/6595/standard/Rune200x200.png?1696506946',
```
Comment (kaladinlight) [inline src/state/slices/opportunitiesSlice/constants.ts:127]:
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```

File: src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Approve.tsx [hunk 9]
```diff
@@ -200,7 +200,7 @@ export const Approve: React.FC<FoxFarmingApproveProps> = ({ accountId, onNext })
       loading={state.loading}
       loadingText={translate('common.approve')}
       preFooter={preFooter}
-      providerIcon='https://assets.coincap.io/assets/icons/256/fox.png'
+      providerIcon='https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025'
```
Comment (kaladinlight) [inline src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Approve.tsx:203]:
```suggestion
      providerIcon='https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025'
```

File: scripts/generateAssetData/overrides.ts [hunk 10]
```diff
@@ -3,7 +3,7 @@ import type { Asset } from '@shapeshiftoss/types'
 
 const fox: Partial<Asset> = {
   color: '#3761F9',
-  icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+  icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```
Comment (kaladinlight) [inline scripts/generateAssetData/overrides.ts:6]:
```suggestion
  icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

File: packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts [hunk 11]
```diff
@@ -232,7 +232,7 @@ describe('cowApi', () => {
               networkName: 'Ethereum',
               precision: 18,
               color: '#5C6BC0',
-              icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
+              icon: '',
```
Comment (kaladinlight) [inline packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts:235]:
```suggestion
              icon: 'https://coin-images.coingecko.com/coins/images/279/large/ethereum.png?1696501628',
```

File: packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts [hunk 12]
```diff
@@ -210,7 +210,7 @@ describe('cowApi', () => {
                     name: 'FOX on Ethereum',
                     precision: 18,
                     color: '#3761F9',
-                    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+                    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```
Comment (kaladinlight) [inline packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts:213]:
```suggestion
                    icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

File: src/contracts/constants.ts (hunk 2 above)
Comment (kaladinlight) [reply src/contracts/constants.ts:1]: You are correct, my bad. As per discussion in dms, will follow up with https://github.com/shapeshift/web/issues/7300, and revert back to the previous import when tackling that issue

Comment (gomesalexandre) [reply src/contracts/constants.ts:1]: Nice one @NeOMakinG! Looks like we've been borked for 2 weeks on regen script too, so this should make things happy again! 🎉 
https://github.com/shapeshift/web/actions/workflows/generate.yml

File: scripts/generateAssetData/generateAssetData.ts (hunk 1 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/generateAssetData.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: scripts/generateAssetData/baseAssets.ts (hunk 3 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/baseAssets.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: scripts/generateAssetData/ethereum/uniswapV2Pools.ts (hunk 4 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/ethereum/uniswapV2Pools.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/plugins/cosmos/index.tsx [hunk 13]
```diff
@@ -12,7 +12,9 @@ export default function register(): Plugins {
       'cosmos:cosmoshub-4',
       {
         name: 'plugins.cosmos.navBar',
-        icon: <AssetIcon src='https://assets.coincap.io/assets/icons/atom@2x.png' />,
+        icon: (
+          <AssetIcon src='https://assets.coingecko.com/coins/images/1481/standard/cosmos_hub.png?1696502525' />
```
[the remark this answers is not in the document]
Comment (NeOMakinG) [reply src/plugins/cosmos/index.tsx]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/state/slices/opportunitiesSlice/constants.ts (hunk 5 above)
Comment (NeOMakinG) [reply src/state/slices/opportunitiesSlice/constants.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/state/slices/opportunitiesSlice/constants.ts (hunk 6 above)
Comment (NeOMakinG) [reply src/state/slices/opportunitiesSlice/constants.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/state/slices/opportunitiesSlice/constants.ts (hunk 7 above)
Comment (NeOMakinG) [reply src/state/slices/opportunitiesSlice/constants.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/state/slices/opportunitiesSlice/constants.ts (hunk 8 above)
Comment (NeOMakinG) [reply src/state/slices/opportunitiesSlice/constants.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Approve.tsx (hunk 9 above)
Comment (NeOMakinG) [reply src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Approve.tsx]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: scripts/generateAssetData/overrides.ts (hunk 10 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/overrides.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts (hunk 11 above)
Comment (NeOMakinG) [reply packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts (hunk 12 above)
Comment (NeOMakinG) [reply packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts]: Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)

File: packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts [hunk 14]
```diff
@@ -232,7 +232,7 @@ describe('cowApi', () => {
               networkName: 'Ethereum',
               precision: 18,
               color: '#5C6BC0',
-              icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
+              icon: 'https://raw.githubusercontent.com/trustwallet/assets/32e51d582a890b3dd3135fe3ee7c20c2fd699a6d/blockchains/ethereum/info/logo.png',
```
Comment (gomesalexandre) [inline packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts:235]: confirmed the intent here is to keep the nicer ETH png

File: packages/swapper/src/swappers/utils/test-data/assets.ts [hunk 15]
```diff
@@ -244,7 +244,7 @@ export const RUNE: Asset = {
   symbol: 'RUNE',
   precision: 8,
   color: '#33FF99',
-  icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+  icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```
Comment (gomesalexandre) [inline packages/swapper/src/swappers/utils/test-data/assets.ts:247]: Not super concerned about this because it's test data, but dis ERC20 RUNE

File: scripts/generateAssetData/baseAssets.ts [hunk 16]
```diff
@@ -109,7 +109,7 @@ export const thorchain: AssetWithNetworkName = {
   symbol: 'RUNE',
   precision: 8,
   color: '#33FF99',
-  icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+  icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:112]: blocking: this is an actual asset used at runtime, and is the wrong icon of ERC-20 RUNE. Can we also use the trust wallet assets URL here?

File: scripts/generateAssetData/baseAssets.ts [hunk 17]
```diff
@@ -53,7 +53,7 @@ export const dogecoin: AssetWithNetworkName = {
   networkName: 'Dogecoin',
   precision: 8,
   color: '#FFC107',
-  icon: 'https://assets.coincap.io/assets/icons/256/doge.png',
+  icon: 'https://assets.coingecko.com/coins/images/5/large/dogecoin.png?1696501409',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:56]: TIL! Looks weird, but is the correct icon 🤯: https://dogecoin.com/trademarks/

File: scripts/generateAssetData/baseAssets.ts [hunk 18]
```diff
@@ -139,7 +139,7 @@ export const bnbsmartchain: AssetWithNetworkName = {
   symbol: 'BNB',
   precision: 18,
   color: '#F0B90B',
-  icon: 'https://rawcdn.githack.com/trustwallet/assets/32e51d582a890b3dd3135fe3ee7c20c2fd699a6d/blockchains/smartchain/info/logo.png',
+  icon: 'https://assets.coingecko.com/coins/images/825/large/bnb-icon2_2x.png?1696501970',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:142]: Confirmed this is the correct one ✅  https://docs.bnbchain.org/bnb-smart-chain/

File: scripts/generateAssetData/ethereum/uniswapV2Pools.ts [hunk 19]
```diff
@@ -105,7 +105,7 @@ export const getUniswapV2Pools = async () => {
         name,
         precision: Number(appTokenData.decimals),
         icons,
-        icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
+        icon: 'https://coin-images.coingecko.com/coins/images/12504/large/uni.jpg?1696512319',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/ethereum/uniswapV2Pools.ts:108]: issue: this doesn't look very asset-y 😞can we ust trust assets here? e.g this guy https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 20]
```diff
@@ -118,13 +118,13 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
...
     provider: DefiProvider.CosmosSdk,
-    icon: 'https://assets.coincap.io/assets/icons/256/atom.png',
+    icon: 'https://coin-images.coingecko.com/coins/images/1481/large/cosmos_hub.png?1696502525',
     color: '#C5B5F2',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.ThorchainSavers]: {
     provider: DefiProvider.ThorchainSavers,
-    icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+    icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```
Comment (gomesalexandre) [inline src/state/slices/opportunitiesSlice/constants.ts:127]: blocking: same issue here

**Review by gomesalexandre — COMMENTED** [review]
A few issues spotted with runtime paths here:

- RUNE icon is the ERC-20 one, not THOR.RUNE
- Wrong brand asset for UNI
- FOX icon *seems* to be an older one, to be confirmed by @reallybeard

File: packages/swapper/src/swappers/utils/test-data/assets.ts (hunk 15 above)
Comment (NeOMakinG) [reply packages/swapper/src/swappers/utils/test-data/assets.ts]: Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)

File: scripts/generateAssetData/baseAssets.ts (hunk 16 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/baseAssets.ts]: Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)

File: scripts/generateAssetData/baseAssets.ts (hunk 17 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/baseAssets.ts:56]: Kept as it is

File: scripts/generateAssetData/ethereum/uniswapV2Pools.ts (hunk 19 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/ethereum/uniswapV2Pools.ts]: Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)

File: src/state/slices/opportunitiesSlice/constants.ts (hunk 20 above)
Comment (NeOMakinG) [reply src/state/slices/opportunitiesSlice/constants.ts]: Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)

**Comment by NeOMakinG** [comment]
@gomesalexandre the asset rebuild is currently running, we will have to push a PR to update the FOX icon on the trustwallet repository

File: scripts/generateAssetData/baseAssets.ts [hunk 21]
```diff
@@ -153,7 +153,7 @@ export const polygon: AssetWithNetworkName = {
   symbol: 'MATIC',
   precision: 18,
   color: '#8f00ff',
-  icon: 'https://assets.coingecko.com/coins/images/4713/large/matic-token-icon.png?1624446912',
+  icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/polygon/info/logo.png',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:156]: That looks weird but that's correct, both variants are valid https://polygontechnology.notion.site/Polygon-Brand-Resources-External-2cd18ae436584e98a6c5aae56db73058?p=aa481b968f2e46c0a83f9da751e0aa17&pm=c

File: scripts/generateAssetData/ethereum/uniswapV2Pools.ts [hunk 22]
```diff
@@ -105,7 +105,7 @@ export const getUniswapV2Pools = async () => {
         name,
         precision: Number(appTokenData.decimals),
         icons,
-        icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
+        icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/ethereum/uniswapV2Pools.ts:108]: Much better!

File: scripts/generateAssetData/overrides.ts [hunk 23]
```diff
@@ -3,7 +3,7 @@ import type { Asset } from '@shapeshiftoss/types'
 
 const fox: Partial<Asset> = {
   color: '#3761F9',
-  icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+  icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/ethereum/assets/0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d/logo.png',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/overrides.ts:6]: Can we add a TODO somewhere either in the codebase or as an issue, to tackle this overide?
Even after the new icon is merged in Coingecko, the override will still apply since we're using `b7a5f12d893fcf58e0eb1dd64478f076857b720b` hash

File: src/state/slices/opportunitiesSlice/constants.ts [hunk 24]
```diff
@@ -118,13 +118,13 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
...
     provider: DefiProvider.CosmosSdk,
-    icon: 'https://assets.coincap.io/assets/icons/256/atom.png',
+    icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/cosmos/info/logo.png',
     color: '#C5B5F2',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.ThorchainSavers]: {
     provider: DefiProvider.ThorchainSavers,
-    icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+    icon: 'https://raw.githubusercontent.com/trustwallet/assets/32e51d582a890b3dd3135fe3ee7c20c2fd699a6d/blockchains/thorchain/info/logo.png',
```
Comment (gomesalexandre) [inline src/state/slices/opportunitiesSlice/constants.ts:127]: here we go!

File: scripts/generateAssetData/overrides.ts (hunk 23 above)
Comment (NeOMakinG) [reply scripts/generateAssetData/overrides.ts]: Not right anymore because of [198a41e](https://github.com/shapeshift/web/pull/7298/commits/198a41ed65fa7b9baf2131ba2eac05808e263b2a)

**Comment by NeOMakinG** [comment]
> Conceptual stamp on this guy pending runtime testing.
> 
> As discussed, we should use githack as a CDN vs. githubusercontent which is _not_ reliable as a CDN for prod, however, since we're going to update the FOX references after the trustwallet/assets PR is merged, happy to have it as a very fast-follow

Yeah this will be a very fast follow

In the meantime, we have a problem with the regeneration, the ETH asset seems like using coincap even after this PR, probably due to `generateRelatedAssetIndex`, couldn't find why yet, still searching

**Review by gomesalexandre — APPROVED** [review]
Tested locally:

- Asset icons are still happy including FOX and UNI

image
image

- DOGE icon is the *akschual* one from brand assets

image

- RUNE icon is still happy

image

- USDC related assets are still happy

image

- FOX related assets are still happy

image

- Re-ran `yarn generate:asset-data` locally and confirmed it didn't produce a weird rugged diff 🎉
